packages/devtools_app/test/debugger_evalution_test.dart

```diff
@@ -23,6 +24,8 @@ void main() {
     const FlutterRunConfiguration(withDebugger: true),
   );
 
+  setGlobal(MessageBus, MessageBus());
```

Comment:
this should be done in flutter_test_environment.dart (see around line 123). then you can remove this from this test and all the other tests below

Reply:
done